pip/cmdoptions.py

```diff
@@ -523,6 +523,56 @@ def only_binary():
 )
 
 
+def strong_hashes():
```

Comment:
In any case, we wouldn't want them to be a set, since the error messages they're used to construct should list the hashes in some kind of order.